tsconfig.json

```diff
@@ -1,14 +1,64 @@
 {
     "compilerOptions": {
-        "target": "es5",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', or 'ESNEXT'. */
-        "allowJs": true,                       /* Allow JavaScript files to be compiled. */
-        "sourceMap": true,                     /* Generates corresponding '.map' file. */
-        "removeComments": false,               /* Do not emit comments to output. */
-        "strict": true,                        /* Enable all strict type-checking options. */
-        "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
-        "noUnusedLocals": true,                /* Report errors on unused locals. */
-        "noUnusedParameters": true,            /* Report errors on unused parameters. */
-        "noImplicitReturns": true              /* Report error when not all code paths in function return a value. */
+        "target": "es5",
+        "allowJs": true,
+        "sourceMap": true,
+        "removeComments": false,
+        "strict": true,
+        "noImplicitAny": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "noImplicitReturns": true,
+        "baseUrl": "./",
+        "paths": {
+            "@pixi.js": ["./bundles/pixi.js"],
+            "@pixi.js-legacy": ["./bundles/pixi.js-legacy"],
+            "@pixi/accessibility": ["./packages/accessibility/src/"],
```

Comment:
Could you provide some more context about why paths to the source file for each package is necessary?

Reply:
Yeah so without the paths typescript doesn't know what these modules are. 

Take this line for example, `import { isMobile } from '@pixi/utils';`. We get an error saying typescript cannot find `@pixi/utils`

By using the paths we map `@pixi/utils` to `../../utils/src/`.

Here's a link to explain how it works [path mapping](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping)

Reply:
If I remove these paths, I am still able to build (`npm run build`) fine without them. In terms of letting packages know about other packages exports, we should do this through module typings (like a very permissive `declare module "@pixi/blah";` in each package or in a global typings folder we can remove later once we emit typings). Shouldn't we be using module resolution in TS to resolve the paths to the packages?

Reply:
If you remove the paths and convert say the utils `index.js` to `index.ts` that is when you get the error.

I'll remove them and keep this pr just for the dependency types